eigrpd/eigrp_main.c

```diff
@@ -100,6 +100,8 @@ static void sigint(void)
 {
 	zlog_notice("Terminating on signal");
 	eigrp_terminate();
+
+	exit(0);
```

Comment:
`EXIT_SUCCESS`?

Reply:
I'm merely following other protocols which exit(0) in signal handling for SIG INT